src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -3052,6 +3052,7 @@ Expression RegularCondition() #RegularCondition:
     | token=<OP_NOTEQUALSSTANDARD> { result = new NotEqualsTo(token.image); }
     | token=<OP_NOTEQUALSBANG> { result = new NotEqualsTo(token.image); }
     | "@@" { result = new Matches(); }
+    | "@?" { result = new Matches(); }
```

Comment:
The problem here is that this statement cannot be reconstructed since a Matches instance is printed as "*@@*". 
BTW what is the sematic of "*@?*"?